content/SCALE/SCALETutorials/Credentials/DirectoryServices/ConfigADSCALE.md

```diff
@@ -26,50 +26,90 @@ Users can configure AD services on Windows or Unix-like operating systems using
 
 To configure an AD connection, you must know the AD controller domain and the AD system account credentials.
 
-### Preparation
+### Preparing to Configure AD in SCALE
 
-Users can take a few steps before configuring Active Directory to ensure the connection process goes smoothly.
+Users can take a few steps before configuring Active Directory (AD) to ensure the connection process goes smoothly.
+
+1. Locate the Relative Distinguished Name (RDN) of the site object. 
+   This is the first component of the distinguishedName property configured in AD.
+   {{< expand "Need AD Help?" "v" >}}
+   To locate the distinguished name: 
+
+   1. Log into AD.
+   2. Click on **View** and select **Advanced Features**.
+   3. Navigate to and right-click on the organizational unit (OU) where you want to read users, service or computer accounts, then select **Properties**.
+   4. Select the **Attribute Editor** tab on the **OU Properties** window.
+   5. Click on **distinguishedName** to highlight it then click **View**.
+   {{< /expand >}}
```

Comment:
What are you trying to do here?

Reply:
Providing basic help with AD not SCALE. If this is not helpful, I'll remove this. If it needs clarification I'll add that.

Reply:
What field are you trying to populate with this information?